website/docs/cloud-docs/run/states.mdx

```diff
@@ -150,7 +170,7 @@ After applying, the run proceeds automatically to completion.
 - If the apply failed, the run ends in the **Apply Errored** state.
 - If a user canceled the apply by pressing the "Cancel Run" button, the run ends in the **Canceled** state.
 
-## 9. Completion
+## 10. Completion
 
 A run is considered completed if it finishes applying, if any part of the run fails, if there's nothing to do, or if a user chooses not to continue. Once a run is completed, the next run in the queue can enter the plan stage.
```

Comment:
```suggestion
A run is complete if it finishes applying, if any part of the run fails, if there is nothing to do, or if a user chooses not to continue. Once a run completes, the next run in the queue can enter the plan stage.
```

Suggest changing "is considered complete" to "is complete" to avoid passive voice style issues. Similarly, suggest changing "is completed" to "completes" to avoid passive voice. Also suggest changing "there's" to "there is" to avoid contractions.